pkg/buildHeaderLineTypeUtilities/buildHeaderLineTypeUtilities.go

```diff
@@ -37,7 +38,38 @@ type HeaderFields struct {
+var (
+	MetHeaderColumnsFileUrl_v12_0 = "https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/data/table_files/met_header_columns_V12.0.txt"
+	MetHeaderColumnsFileUrl_v11_1 = "https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/data/table_files/met_header_columns_V11.1.txt"
+	MetHeaderColumnsFileUrl_v11_0 = "https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/data/table_files/met_header_columns_V11.0.txt"
+	MetHeaderColumnsFileUrl_v10_1 = "https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/data/table_files/met_header_columns_V10.1.txt"
+	MetHeaderColumnsFileUrl_v10_0 = "https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/data/table_files/met_header_columns_V10.0.txt"
+	MetHeaderColumnsFileUrl       = MetHeaderColumnsFileUrl_v12_0
+)
+
+var MetUserDocFiles = []string{
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/ensemble-stat.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/grid-stat.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/gsi-tools.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/mode-td.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/mode.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/point-stat.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/stat-analysis.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/tc-gen.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/tc-pairs.rst",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/docs/Users_Guide/wavelet-stat.rst",
+}
+
+var MetSrcFiles = []string{
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/libcode/vx_analysis_util/mode_line.cc",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/libcode/vx_grid/unstructured_grid.cc",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/libcode/vx_tc_util/prob_info_base.cc",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/libcode/vx_tc_util/prob_rirw_info.cc",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/libcode/vx_tc_util/prob_rirw_pair_info.cc",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/libcode/vx_tc_util/track_pair_info.cc",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/tools/core/stat_analysis/parse_stat_line.cc",
+	"https://raw.githubusercontent.com/dtcenter/MET/refs/heads/main_v12.0/src/tools/tc_utils/tc_stat/tc_stat_job.cc",
+}
```

Comment:
I wanted to double check two things here:

1. Should these files lists have a version switch built in? Or will we update them to e.g. - `main_v13.0` when MET 13 comes out?
2. Do we want to be relying on the release branch or on particular release tags for these files? It looks like the `main_v12.0` branch will be silently updated for each "patch release" tag so running the generator will always get the latest. I suspect that's what we want. If it's not, relying on tags like https://raw.githubusercontent.com/dtcenter/MET/refs/tags/v12.0.2/docs/Users_Guide/ensemble-stat.rst will get particular patch releases.

Reply:
I intended that we would always use the latest version whenever we upgrade. This is actually a temporary solution anyway. By the next major METplus release I intend to have a single file from which we get the datatypes instead of scraping src files and user docs.

Reply:
Sounds great. Thanks, Randy!